project.clj

```diff
@@ -28,7 +28,7 @@
                  [org.cyverse/metadata-client "3.0.1"]
                  [org.cyverse/common-cli "2.8.1"]
                  [org.cyverse/common-cfg "2.8.1"]
-                 [org.cyverse/common-swagger-api "2.10.2"]
+                 [org.cyverse/common-swagger-api "2.10.3-SNAPSHOT"]
```

Comment:
I just published version `2.10.3`, so this might have to be bumped to `2.10.4-SNAPSHOT`.